main.java

```diff
@@ -11,64 +13,99 @@ public static void main(String[] args) {
 
         System.out.println("Choose your magical creature to use for battle!");
         System.out.println("-".repeat(93));
-        //list of creatures
-        System.out.println("Troll");
-        System.out.println("Dragon");
-        System.out.println("Hippogriff");
-        System.out.println("Unicorn");
-        System.out.println("Cerberus");
-        System.out.println("Mandrake");
+         
+     //creating the creatures 
+        var myDragon = new Dragon();
+        var myHippogriff = new Hippogriff();
+        var myMandrake = new Mandrake();
+        var myThreeHeadedDog = new ThreeHeadedDog();
+        var myTroll = new Troll();
+        var myUnicorn = new Unicorn();
+
+    //make array of creatures to number
+      ArrayList<String> Creat = new ArrayList<String>();  
+        Creat.add("Dragon (0)");
+        Creat.add("Hippogriff (1)");
+        Creat.add("Mandrake (2)");
+        Creat.add("ThreeHeadedDog (3)");
+        Creat.add("Troll (4)");
+        Creat.add("Unicorn (5)");
+        System.out.println(Creat);
+
 
 //user input
     System.out.println();
     Scanner myObj = new Scanner(System.in);
     
     String creatureName1;
     // Enter first creature and press Enter
-    System.out.println("Enter first creature: "); 
+    System.out.println("Enter first creature using the corresponding number that belongs to the creature you want: "); 
     creatureName1 = myObj.nextLine();   
```

Comment:
then the user selects the creature by an index number.
```suggestion
    selectedCreatureIndex = myObj.nextInt();   // or something like that
    
    // Then you can find the creatre in the array based on the selected index
    selectedCreature = availableCreatures[selectedCreatureIndex];
```